src/java/org/jivesoftware/openfire/roster/RosterManager.java

```diff
@@ -712,32 +723,28 @@ private void groupUserDeleted(Group group, Collection<JID> users, JID deletedUse
         // Get the roster of the deleted user.
         Roster deletedUserRoster = null;
         if (server.isLocal(deletedUser)) {
-            deletedUserRoster = rosterCache.get(deletedUser.getNode());
+            try {
+                deletedUserRoster = getRoster(deletedUser.getNode());
+            } catch (UserNotFoundException e) {
+            }
```

Comment:
Please don't catch exceptions without doing anything with them. Even if these exceptions cannot happen, chances are that some future code change might (by accident) cause them to trigger. At the very least log an exception, but if the exception really is "impossible", apply the fail-fast strategy: rethrow it, or wrap it in a runtime exception that's thrown.